src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/Cas2ApplicationEntity.kt

```diff
@@ -68,6 +86,32 @@ ORDER BY createdAt DESC
   fun findSubmittedCas2ApplicationSummariesCreatedByUser(userId: UUID, pageable: Pageable?):
     Page<Cas2ApplicationSummary>
 
+  @Query(
+    """
+SELECT
+    CAST(a.id AS TEXT) as id,
+    a.crn,
+    CAST(a.created_by_user_id AS TEXT) as createdByUserId,
+    a.created_at as createdAt,
+    a.submitted_at as submittedAt,
+    asu.label as latestStatusUpdateLabel,
+    CAST(asu.status_id AS TEXT) as latestStatusUpdateStatusId
+FROM cas_2_applications a
+LEFT JOIN
```

Comment:
hmm interestingly this LEFT JOIN query works with pagination but the one for `findSubmittedCas2ApplicationSummariesCreatedByUser` doesn't work when a page param is given 🤯 , which is why I've proposed this change https://github.com/ministryofjustice/hmpps-approved-premises-api/pull/1752

I have no idea what's going on so will bring to dev huddle